src/task_run_diagnostic_reports.py

```diff
@@ -9,7 +9,12 @@
 def run_diagnostic_reports(group, disease):
     config = Config()
     reports = config.diagnostic_reports(group, disease)
-    return run_reports(config, reports)
+    if len(reports) > 0:
```

Comment:
Regarding your question of what happens if we trigger the task for an unknown disease or group, it would just find no configuration and would previously silently do nothing. So I've added some logging here. 

If there's config, but the report can't run because the group or disease, or the report, isn't set up at the Orderly end: as you say that will be logged as an error, either on the first request to run, or later during polling (separate ticket).